plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryQueryRunner.java

```diff
@@ -83,6 +89,8 @@ public static Session createSession()
     public static class BigQuerySqlExecutor
             implements SqlExecutor
     {
+        private static final Map.Entry<String, String> BIG_QUERY_SQL_EXECUTOR_LABEL = Maps.immutableEntry("ci-automation-source", "big_query_sql_executor");
```

Comment:
nit: let's use a more Trino specific name. Some people might be running BigQuery tests in their GCP accounts and it'll be easier to know who is adding these labels.

Reply:
Meh, I don't have opinions on this one. The existing one looks better than whatever I'd come up with.